diff --git a/dbsake/cli/cmd/sandbox.py b/dbsake/cli/cmd/sandbox.py
index b049718..297c5f3 100644
--- a/dbsake/cli/cmd/sandbox.py
+++ b/dbsake/cli/cmd/sandbox.py
@@ -82,7 +82,7 @@ def sandbox_cli(sandbox_directory,
     """Create a sandboxed MySQL instance.
 
     This command installs a new MySQL instance under the specified sandbox
-    directory, or under ~/sandboxes/sandbox_<datetime> if none is specified.
+    directory, or under ~/.dbsake/sandboxes/sandbox_<datetime> if none is specified.
     """
     from dbsake.core.mysql import sandbox
 
diff --git a/dbsake/core/mysql/sandbox/common.py b/dbsake/core/mysql/sandbox/common.py
index 31607d1..13c30ea 100644
--- a/dbsake/core/mysql/sandbox/common.py
+++ b/dbsake/core/mysql/sandbox/common.py
@@ -52,6 +52,7 @@ def check_options(**kwargs):
     basedir = kwargs.pop('sandbox_directory')
     if not basedir:
         basedir = os.path.join('~',
+                               '.dbsake',
                                'sandboxes',
                                'sandbox_' + time.strftime('%Y%m%d_%H%M%S'))
     basedir = os.path.abspath(os.path.expanduser(basedir))
diff --git a/docs/commands/sandbox.rst b/docs/commands/sandbox.rst
index 54b7724..2bc5efb 100644
--- a/docs/commands/sandbox.rst
+++ b/docs/commands/sandbox.rst
@@ -5,7 +5,7 @@ sandbox
 
 Setup a secondary MySQL instance painlessly.
 
-This setups a MySQL under ~/sandboxes/ (by default) with a
+This setups a MySQL under ~/.dbsake/sandboxes/ (by default) with a
 randomly generated password for the root@localhost user
 and networking disabled.
 
