diff --git a/pext/__main__.py b/pext/__main__.py
index de78a34..7ae28b5 100644
--- a/pext/__main__.py
+++ b/pext/__main__.py
@@ -3313,5 +3313,9 @@ def main(ui_type: UIType) -> None:
     main_loop.run()
 
 
+def entry_main() -> None:
+    main(UIType.Qt5)
+
+
 if __name__ == "__main__":
     main(UIType.Qt5)
diff --git a/pext/ui/qt5.py b/pext/ui/qt5.py
index c942615..9486458 100644
--- a/pext/ui/qt5.py
+++ b/pext/ui/qt5.py
@@ -47,7 +47,7 @@ from PyQt5.Qt import (QClipboard, QObject, QStringListModel, QQmlApplicationEngi
                       QQmlProperty, QUrl)
 from PyQt5.QtGui import QWindow
 
-from __main__ import (AppFile, ConfigRetriever, Core, InternalCallProcessor, LocaleManager, Logger, ModuleManager,
+from pext.__main__ import (AppFile, ConfigRetriever, Core, InternalCallProcessor, LocaleManager, Logger, ModuleManager,
                       MinimizeMode, OutputMode, OutputSeparator, ProfileManager, RunConseq, Settings, SortMode,
                       ThemeManager, Translation, UiModule, UpdateManager)
 from constants import USE_INTERNAL_UPDATER
diff --git a/requirements.txt b/requirements.txt
index 896d6d8..7a18118 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,9 +1,9 @@
-dulwich==0.20.20
-paramiko==2.7.2
-psutil==5.8.0
-pynput==1.7.3
-pyqt5==5.15.2
-requests==2.25.1
-watchdog==2.0.2
-PyAutoGUI==0.9.52; sys.platform == 'darwin'
-PyOpenGL==3.1.5; sys.platform == 'linux'
+dulwich
+paramiko
+psutil
+pynput
+pyqt5
+requests
+watchdog
+PyAutoGUI; sys.platform == 'darwin'
+PyOpenGL; sys.platform == 'linux'
diff --git a/setup.py b/setup.py
index f49373d..1082aee 100644
--- a/setup.py
+++ b/setup.py
@@ -76,6 +76,7 @@ setup(
     packages=[
         'pext',
         'pext/helpers',
+        'pext/ui',
         'pext_dev'
     ],
     package_data={'pext': ['VERSION', 'i18n/*.qm', 'images/128x128/*', 'images/scalable/*', 'qml/*', 'helpers/*', '*.py', 'Pext.workflow/*'],
@@ -83,7 +84,7 @@ setup(
     zip_safe=False,
     entry_points={
         'gui_scripts': [
-            'pext=pext.__main__:main'
+            'pext=pext.__main__:entry_main'
         ],
         'console_scripts': [
             'pext_dev=pext_dev.__main__:main'
