diff --git a/m4/guestfs_appliance.m4 b/m4/guestfs_appliance.m4
--- a/m4/guestfs_appliance.m4
+++ b/m4/guestfs_appliance.m4
@@ -106,6 +106,8 @@ elif test -f /etc/debian_version; then
     if grep -q 'DISTRIB_ID=Ubuntu' /etc/lsb-release 2>&AS_MESSAGE_LOG_FD; then
         DISTRO=UBUNTU
     fi
+elif test -f /etc/gentoo-release; then
+    DISTRO=GENTOO
 elif test -f /etc/arch-release; then
     DISTRO=ARCHLINUX
 elif test -f /etc/SuSE-release; then
