diff --git a/m4/guestfs_misc.m4 b/m4/guestfs_misc.m4
--- a/m4/guestfs_misc.m4
+++ b/m4/guestfs_misc.m4
@@ -19,8 +19,9 @@ dnl Miscellaneous configuration that doesn't fit anywhere else.

 dnl Replace libtool with a wrapper that clobbers dependency_libs in *.la files
 dnl http://lists.fedoraproject.org/pipermail/devel/2010-November/146343.html
-LIBTOOL='bash $(top_srcdir)/libtool-kill-dependency_libs.sh $(top_builddir)/libtool'
-AC_SUBST([LIBTOOL])
+dnl LIBTOOL='bash $(top_srcdir)/libtool-kill-dependency_libs.sh $(top_builddir)/libtool'
+dnl AC_SUBST([LIBTOOL])
+LT_INIT

 dnl Only build boot-analysis program on x86-64 and aarch64.  It
 dnl requires custom work to port to each architecture.
